### PR TITLE
Updated SettingsSchema.md with default values for global and profiles

### LINKED ISSUE #1712: Documentation Issue: optional values SettingsSchema should list default value
Any optional settings in [SettingsSchema.md](https://github.com/microsoft/terminal/blob/master/doc/cascadia/SettingsSchema.md#globals) should have their default value listed. Otherwise, it may require some trial-and-error to determine which setting you are currently using.

A change like #918 then could have update the default values in the docs.

### LINKED ISSUE #1753: Add setting to control Acrylic "Blur" in addition to opacity
Pretty much the title.

Right now we're only using the default blur amount.

### PR DESCRIPTION
**Pull request by cinnamon-msft**
## Summary of the Pull Request
Added default values for global and profiles within SettingsSchema.md

 
## References
#1712 

## PR Checklist
* [x] Closes #1712 
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

### CODE REVIEW

File: doc/cascadia/SettingsSchema.md [hunk 1]
```diff
@@ -3,43 +3,43 @@
...
+| `colorScheme` | _Required_ | String | `Campbell` | Name of the terminal color scheme to use. Color schemes are defined under `schemes`. |
+| `commandline` | _Required_ | String | `powershell.exe` | Executable used in the profile. |
+| `cursorColor` | _Required_ | String | `#FFFFFF` | Sets the cursor color for the profile. Uses hex color format: `"#rrggbb"`. |
+| `cursorShape` | _Required_ | String | `bar` | Sets the cursor shape for the profile. Possible values: `"vintage"` ( &#x2583; ), `"bar"` ( &#x2503; ), `"underscore"` ( &#x2581; ), `"filledBox"` ( &#x2588; ), `"emptyBox"` ( &#x25AF; ) |
+| `fontFace` | _Required_ | String | `Consolas` | Name of the font face used in the profile. |
+| `fontSize` | _Required_ | Integer | `10` | Sets the font size. |
+| `guid` | _Required_ | String | | Unique identifier of the profile. Written in registry format: `"{00000000-0000-0000-0000-000000000000}"`. |
+| `historySize` | _Required_ | Integer | `9001` | The number of lines above the ones displayed in the window you can scroll back to. |
+| `name` | _Required_ | String | `PowerShell Core` | Name of the profile. Displays in the dropdown menu. |
```
Comment (DHowett-MSFT) [inline doc/cascadia/SettingsSchema.md]: There is no default here really

Comment (elrussord1) [reply doc/cascadia/SettingsSchema.md]: UPDATING SISTEM FOR CC CHRT WALTER 
ELRUSSORD*

**Comment by Summon528** [comment]
How does this close #1753 though

**Comment by cinnamon-msft** [comment]
> How does this close #1753 though

Oops, you're right! Fixing issue number now 😊

File: doc/cascadia/SettingsSchema.md (hunk 1 above)
Comment (cinnamon-msft) [reply doc/cascadia/SettingsSchema.md]: Should I keep PowerShell guid in globals for defaultProfile?

**Pull request merged**